src/main/java/com/nitnelave/CreeperHeal/block/CreeperContainer.java

```diff
@@ -0,0 +1,140 @@
+package com.nitnelave.CreeperHeal.block;
+
+import com.nitnelave.CreeperHeal.config.CfgVal;
+import com.nitnelave.CreeperHeal.config.CreeperConfig;
+import com.nitnelave.CreeperHeal.config.WCfgVal;
+import org.bukkit.Chunk;
+import org.bukkit.Location;
+import org.bukkit.World;
+import org.bukkit.block.BlockState;
+import org.bukkit.block.Container;
+import org.bukkit.inventory.DoubleChestInventory;
+import org.bukkit.inventory.Inventory;
+import org.bukkit.inventory.ItemStack;
+
+import java.util.Collection;
+import java.util.concurrent.ThreadLocalRandom;
+
+/**
+ * InventoryHolder implementation of CreeperMultiblock.
+ *
+ * @author Jikoo
+ */
+class CreeperContainer extends CreeperMultiblock
+{
+
+    /*
+     * Constructor.
+     */
+    CreeperContainer(Container blockState)
+    {
+        super(blockState);
+
+        Inventory inv = blockState.getInventory();
+        if (!(inv instanceof DoubleChestInventory))
+            return;
```

Comment:
Rather than putting a return, in the case of a constructor, I'd rather put the rest of the function inside the `if`. It strikes me as clearer.